### ISSUE TITLE
DatabaseMetadata.getColumns doesn't returns IS_AUTOINCREMENT

### ISSUE THREAD

**Issue by argan** [xerial/sqlite-jdbc#132]
for version 3.8.11.2,there is no 'IS_AUTOINCREMENT' returned.

`        //        IS_AUTOINCREMENT String => Indicates whether this column is auto incremented
        //        YES --- if the column is auto incremented
        //        NO --- if the column is not auto incremented
        //        empty string --- if it cannot be determined whether the column is auto incremented parameter is unknown
        checkOpen();

```
    StringBuilder sql = new StringBuilder(700);
    sql.append("select null as TABLE_CAT, null as TABLE_SCHEM, tblname as TABLE_NAME, ")
       .append("cn as COLUMN_NAME, ct as DATA_TYPE, tn as TYPE_NAME, 2000000000 as COLUMN_SIZE, ")
       .append("2000000000 as BUFFER_LENGTH, 10   as DECIMAL_DIGITS, 10   as NUM_PREC_RADIX, ")
       .append("colnullable as NULLABLE, null as REMARKS, colDefault as COLUMN_DEF, ")
       .append("0    as SQL_DATA_TYPE, 0    as SQL_DATETIME_SUB, 2000000000 as CHAR_OCTET_LENGTH, ")
       .append("ordpos as ORDINAL_POSITION, (case colnullable when 0 then 'NO' when 1 then 'YES' else '' end)")
       .append("    as IS_NULLABLE, null as SCOPE_CATLOG, null as SCOPE_SCHEMA, ")
       .append("null as SCOPE_TABLE, null as SOURCE_DATA_TYPE from (");`
```

**Comment by sualeh** [comment]
Please close this issue, @xerial

**Comment by sualeh** [comment]
Fixed in 3.20.1
